distribution/docs/src/main/resources/_contents/_developing/developing-intro-contents.adoc

```diff
@@ -0,0 +1,816 @@
+
+    <configfile finalname="/data/solr/metacard_cache/conf/solrconfig.xml">mvn:ddf.platform.solr/platform-solr-server-standalone/${ddf.version}/xml/solrconfig</configfile>
+    <configfile finalname="/data/solr/metacard_cache/conf/schema.xml">mvn:ddf.platform.solr/platform-solr-server-standalone/${ddf.version}/xml/schema</configfile>
+    <configfile finalname="/data/solr/metacard_cache/conf/protwords.txt">mvn:ddf.platform.solr/platform-solr-server-standalone/${ddf.version}/txt/protwords</configfile>
+    <configfile finalname="/data/solr/metacard_cache/conf/stopwords_en.txt">mvn:ddf.platform.solr/platform-solr-server-standalone/${ddf.version}/txt/stopwords_en</configfile>
+    <configfile finalname="/data/solr/metacard_cache/conf/stopwords.txt">mvn:ddf.platform.solr/platform-solr-server-standalone/${ddf.version}/txt/stopwords</configfile>
+    <configfile finalname="/data/solr/metacard_cache/conf/synonyms.txt">mvn:ddf.platform.solr/platform-solr-server-standalone/${ddf.version}/txt/synonyms</configfile>
+</feature>
+----
+
+==== Making Sure a Features File Will Display Properly in the Installer
+
+In order to ensure that the installer can correctly interpret and display application details, there are several guidelines that should be followed when creating the features file for the application.
+
+* Be sure that only one feature in the `features.xml` has the `auto-install` tag.
++
+[source]
+----
+install='auto'
+----
++
+This is the feature that the installer displays to the user (name, description, version, etc.). It is typically named after the application itself and the description provides a complete application description.
+* Be sure that the one feature specified to `auto-install` has a complete list of all of its dependencies in order to ensure the dependency tree can be constructed correctly.
+
+===== Auto-starting an Application Feature
+
+Within the `features.xml` file for an application, one feature will have the install attribute set to `auto`.
+Within this feature, refer to any dependencies of the application as well as any features that should start automatically.
+Other features should have install set to `manual`.
+
+The following example demonstrates configuring features to be auto-started. The naming convention for this feature is typically "`application name`" + "`-app`," as shown.
+
+.Auto-start features
+[source,xml,linenums]
+----
+<feature name="catalog-app" install="auto">
+    <feature>platform-app</feature>
+    <feature>catalog-core</feature>
+    <feature>catalog-core-metricsplugin</feature>
+    <feature>catalog-core-sourcemetricsplugin</feature>
+    <feature>catalog-transformer-thumbnail</feature>
+</feature>
+----
+
+=== Developing ${branding} Applications
+
+The ${branding} applications are comprised of components, packaged as Karaf features, which are collections of OSGi bundles.
+These features can be installed/uninstalled using the ${admin-console} or ${command-console}.
```

Comment:
Ah. Wait. Does the `admin-console` replaceable param refer to our Admin console or to the Felix Web Console? If the former, than my earlier comments about it should be updated. If it's our Admin console, then we probably shouldn't talk about it earlier as being provided by Karaf.